### ISSUE TITLE
[BUG] Discrim Does Not Show 0's

### ISSUE THREAD

**Issue by ImmuneLion318** [Lachee/discord-rpc-csharp#162]
Uh So After Initialization Of The Presence During The OnReady Event I Send To My Websocket CurrentUser.Username And Discriminator But The Discriminator Dosen't Appear To Show Any Of The 0's So My User Comes Up As {User}1 Rather Than The Appropriate Format Of {User}0001 Any Help Would Be Appreciated

**Comment by Lachee** [comment]
Use the [User.ToString](https://lachee.github.io/discord-rpc-csharp/docs/api/DiscordRPC.User.html#DiscordRPC_User_ToString) function to display a user as `username#discrim` just like discord. 

If you only want the formatted discrim, look at the [Source Code](https://github.com/Lachee/discord-rpc-csharp/blob/master/DiscordRPC/User.cs/#L224-L227) for that function.

TL;DR its a standard C# formatting you can apply to the integer to get "left padding"
